app/views/users/show.html.erb

```diff
@@ -70,10 +70,12 @@
           <div class="profile-edit-button">
```

Comment:
In the user info box, would be great if you could display whether a user wanted to "volunteer" or is "looking for volunteers". I think just focus on getting the information in the box, and we can worry about styling later (aka grace lol) 

But like we discussed a while ago, have the name + role (volunteer vs looking) centered beneath the profile picture!

![screen shot 2017-04-04 at 3 21 39 am](https://cloud.githubusercontent.com/assets/10657390/24652267/0f653154-18e6-11e7-9564-c7be343bd3ca.png)